src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRowLabel.js

```diff
@@ -7,7 +7,7 @@ const EntryRowLabel = ({ color, icon, text }) => {
     <Label
       color={color}
       icon={icon}
-      style={{ marginTop: '16px', marginBottom: '4px' }}
+      style={{ marginTop: '0.25rem', marginBottom: '0.25rem' }}
```

Comment:
Even better approach would be to use PF variables or classes to position the element if possible. More info: https://www.patternfly.org/design-foundations/spacers

Reply:
Done and thanks!  I didn't know about them before now.